File: trunk/java/formulario-integrado/src/formulario/integrado/model/Tipo.java
package formulario.integrado.model;

public class Tipo extends Model {
    
    private int id;
    private String descricao;

    /**
     * @return the id
     */
    public int getId() {
        return id;
    }

    /**
     * @param id the id to set
     */
    public void setId(int id) {
        this.id = id;
    }

    /**
     * @return the descricao
     */
    public String getDescricao() {
        return descricao;
    }

    /**
     * @param descricao the descricao to set
     */
    public void setDescricao(String descricao) {
        this.descricao = descricao;
    }

    @Override
    public boolean validate() {
        if (this.descricao.isEmpty() || this.descricao == null ) {
            super.addErrors("descricao");
        } else {
            if (this.descricao.length() > 10) {
                super.addErrors("descricao");
            }
        }
        
        return super.getErrors().isEmpty();
    }
}

File: trunk/web/application/models/Tipo.php
<?php
require_once(APPLICATION . '/models/Model.php');

class Tipo extends Model {

    private $id;
    private $descricao;

    public function getId() {
        return $this->id;
    }

    public function setId($id) {
        return $this->id = $id;
    }

    public function getDescricao() {
        return $this->descricao;
    }

    public function setDescricao($descricao) {
        return $this->descricao = $descricao;
    }

}